Assets/VRTK/Scripts/Locomotion/VRTK_GazeDrivenLocomotion.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Gaze Driven Movement|Scripts|0070
+namespace VRTK
+{
+    using UnityEngine;
+    using UnityEngine.UI;
+    /// <summary>
+    /// The Gaze Driven Movement script creations a directional "joystick" underneath a given game object within the seen. Movement is driven by moving the joystick using the HMD to drag across the gamepad.
+    /// </summary>
+    /// <remarks>
+    /// This script is meant to be used for 3rd person VR games.
+    /// Attach this script to the `[CameraRig]` prefab. Attach the VRTK_DirectionPad prefab to get the UI for the directional pad.
+    /// </remarks>
+    public class VRTK_GazeDrivenLocomotion: MonoBehaviour
+    {
+        [Tooltip("The speed of the character when HMD gaze is at the edge of the game pad")]
+        [SerializeField]
+        private float maxSpeed = 2.0f;
+        [Tooltip("The directionpad UI that will show the user where their gaze is and how it's affected the locomotion of the character. Use the DirectionPad Prefab in VRTK/Prefabs")]
+        [SerializeField]
+        private GameObject directionPad;
+        [Tooltip("The character game object that you want to locomote.")]
+        [SerializeField]
+        private GameObject characterModel;
+        [Tooltip("Sets the size of the gamepad that spawns underneath the player, which affects the sensitivity and where users look to drive movement of character")]
+        [SerializeField]
+        private float gamePadSize;
+        [Tooltip("The character should be attached to an empty game object, so that certain objects that should not rotate with the character can be attached the empty root object")]
+        [SerializeField]
+        private GameObject characterController;
+        private Quaternion characterLookRotation;
+        private Vector3 directionVector = new Vector3(0, 0, 0);
+        private GameObject interactionPlane;
+        private Slider slider;
+        private Transform hmd; 
+        private void Start()
```

Comment:
Also, for neatness, add a line break between the variable declaration and the method declaration.